### PR TITLE
Issue 292 : Create schema and add instances for body substance (BodySubstance)

### PR DESCRIPTION
**Pull request by ehennestad**

### CODE REVIEW

**Comment by lzehl** [comment]
@ehennestad this PR includes too many unrelated changes. Could you please fix this?

File: instances/bodySubstance/blood.jsonld [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "@context": {
     "@vocab": "https://openminds.ebrains.eu/vocab/"
   },
-  "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/blood",
-  "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+  "@id": "https://openminds.ebrains.eu/instances/bodySubstance/blood",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySubstance",
   "definition":  "'Blood' is a body fluid. It runs through the circulatory system of vertebrates and delivers necessary substances such as nutrients and oxygen to the cells, and transports metabolic waste products away from those same cells.",
```
Comment (UlrikeS91) [inline instances/bodySubstance/blood.jsonld]: "'Blood' is a body fluid in the circulatory system of vertebrates that transports substances to and from cells (e.g. nutrients, oxygen or metabolic waste products). [adapted from [Wikipedia](https://en.wikipedia.org/wiki/Blood)]"

File: instances/bodySubstance/blood.jsonld [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
   "description": "A bodily fluid that is composed of blood plasma and erythrocytes (blood cells).",
```
Comment (UlrikeS91) [inline instances/bodySubstance/blood.jsonld:8]: "Blood is a body fluid composed of four main components: plasma, erythrocytes (red blood cells), leukocytes (white blood cells) and thrombocytes (platelets). Its function in the circulatory system is to deliver nutrients and oxygen to cells and transport substances such as metabolic waste products away from the cells."

File: instances/terminology/bodySubstance.jsonld [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/terminology/bodySubstance",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Terminology",
+  "definition": "",
+  "description": "Material anatomical entity in a gaseous, liquid, semisolid or solid state; produced by anatomical structures or derived from inhaled and ingested substances that have been modified by anatomical structures as they pass through the body. The preferred ontology for 'body substance' is UBERON.",
```
Comment (UlrikeS91) [inline instances/terminology/bodySubstance.jsonld]: "Any material anatomical entity in a gaseous, liquid, semisolid or solid state produced by or derived from an organism or parts of an organism."

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/terminology/bodySubstance.jsonld (hunk 3 above)
Comment (lzehl) [reply instances/terminology/bodySubstance.jsonld]: put this suggestion as definition. leave description null

**Review by lzehl — CHANGES_REQUESTED** [review]
general comment:

In openMINDS, the controlled terms should have a one (max two sentence) "definition" and if you want to you can give a longer more elaborate "description". Please set "description" to null if it is not more elaborate than the "definition".

File: instances/bodySubstance/blood.jsonld (hunk 2 above)
Comment (ehennestad) [reply instances/bodySubstance/blood.jsonld:8]: Updated in 917ed9b0c841a8517b62b18aa4646b12b3099338

File: instances/terminology/bodySubstance.jsonld (hunk 3 above)
Comment (ehennestad) [reply instances/terminology/bodySubstance.jsonld]: Updated in ac187e1c54d6457a62a61d84989d80de12edde47

File: instances/bodySubstance/blood.jsonld [hunk 4]
```diff
@@ -2,9 +2,9 @@
   "@context": {
     "@vocab": "https://openminds.ebrains.eu/vocab/"
   },
-  "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/blood",
-  "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
-  "definition":  "'Blood' is a body fluid. It runs through the circulatory system of vertebrates and delivers necessary substances such as nutrients and oxygen to the cells, and transports metabolic waste products away from those same cells.",
+  "@id": "https://openminds.ebrains.eu/instances/bodySubstance/blood",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySubstance",
+  "definition": "'Blood' is a body fluid in the circulatory system of vertebrates that transports substances to and from cells (e.g. nutrients, oxygen or metabolic waste products). [adapted from Wikipedia]",
```
Comment (UlrikeS91) [inline instances/bodySubstance/blood.jsonld]: "adapted from Wikipedia" should be a hyperlink to https://en.wikipedia.org/wiki/Blood

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/terminology/bodySubstance.jsonld [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  "@id": "https://openminds.ebrains.eu/instances/terminology/bodySubstance",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Terminology",
+  "definition": "Any material anatomical entity in a gaseous, liquid, semisolid or solid state produced by or derived from an organism or parts of an organism.",
+  "description": "The preferred ontology for 'body substance' is UBERON.",
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "body substance",
+  "preferredOntologyIdentifier": null,
+  "synonym": null
+}
```
Comment (lzehl) [inline instances/terminology/bodySubstance.jsonld:14]: new line at the end is missing

File: instances/bodySubstance/arterialBlood.jsonld [hunk 6]
```diff
@@ -2,8 +2,8 @@
   "@context": {
     "@vocab": "https://openminds.ebrains.eu/vocab/"
   },
-  "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/arterialBlood",
-  "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+  "@id": "https://openminds.ebrains.eu/instances/bodySubstance/arterialBlood",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySubstance",
```
Comment (lzehl) [inline instances/bodySubstance/arterialBlood.jsonld:6]: based on our discussion in #292 this should be OrganismSubstance

**Review by lzehl — CHANGES_REQUESTED** [review]
based on our discussion in https://github.com/HumanBrainProject/openMINDS_controlledTerms/issues/292 we should use OrganismSubstance and not BodySubstance.

@ehennestad could you update and resolve the conflicts for this PR ?

**Pull request closed without merging**